modules/ROOT/pages/dw-core-functions-ceil.adoc

```diff
@@ -3,35 +3,48 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-//* <<ceil1>>
-
 
 [[ceil1]]
 == ceil(Number): Number
 
-Rounds a number upwards, returning the first full number above than the
-one provided.
+Rounds an input number up to the nearest whole number.
+
+
+=== Parameters
+
+[%header, cols="1,3"]
+|===
+| Name   | Description
+| `number` | The number to round.
```

Comment:
The number to round *up*. ?

Reply:
Also can this be an array or must it? If support of array vs. single number is perfectly consistent across all functions we can just say it once, but mentioning it sometimes and not others can be confusing.